README.md

```diff
@@ -54,13 +54,9 @@ After a brief frontend load time, the app should be accessible at https://localh
 
 Good for <10k sessions and <50k errors ingested monthly. See our  [docs for more info and limitations](https://www.highlight.io/docs/general/company/open-source/hosting/self-host-hobby).
 
-### Developing on Highlight
-
-Want to contribute to Highlight? See our [onboarding](./DEVELOPER_ONBOARDING.md) guide.
-
```

Comment:
we have a guide for this too https://www.highlight.io/docs/getting-started/self-host/dev-deployment-guide

Reply:
This was actually something more in depth for internal Highlight developers: https://github.com/highlight/highlight/commit/ca73601ffee731183c6c693d7f9ede9d35b90935

The file was deleted in this [commit](https://github.com/highlight/highlight/commit/0496ceafb0a9cdbc7f08cb50d4a9d349490ba1bd). 

Would rather reduce the amount of linking we have to do between GitHub READMEs as it results in a lot of manual work when things break.